install_jdk_and_maven

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright 2019-2020 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+# Install OS packages that support most software we build
+# * openjdk15-jdk: smaller than openjdk15, which includes docs and demos
+# * openjdk15-jmods: needed for module support
+# * binutils: needed for some node modules and jlink --strip-debug
+# * tar: BusyBux built-in tar doesn't support --strip=1
+
+set -uex
+
+# Ensure we can create hs_err_pid*.log
+ulimit -c unlimited
+function maybe_log_crash() {
+  (cat $(ls hs_err_pid*.log) 2>&- || true) && exit 1;
```

Comment:
ps this works as it crashes on s390x and exits after logging. I'll rebase the other PR #52 after merge